action.yml

```diff
@@ -117,11 +117,21 @@ runs:
             fi
 
             if [[ "${{ env.PRIORITY }}" == "v2" ]]; then
-              echo "url=$(cd oxygen_v2 && ln -s ../node_modules node_modules && $oxygen_v2_command --buildCommand="${build_command_filtered}" )" >> $GITHUB_OUTPUT
+              output=$(cd oxygen_v2 && ln -s ../node_modules node_modules && $oxygen_v2_command --buildCommand="${build_command_filtered}")
+              exit_code=$?
+
+              if [ $exit_code -eq 0 ]; then
+                echo "url=$output" >> $GITHUB_OUTPUT
+              else
+                exit 1
+              fi
```

Comment:
This could potentially be cleaned up. I want to ensure I'm getting the actual exit code of the command itself, not the echo that we do to dump into `$GITHUB_OUTPUT`.